app/components/global-search-box.hbs

```diff
@@ -1,6 +1,7 @@
 <div class="global-search-box" data-test-global-search-box>
   <input
     aria-label={{t "general.searchTheCurriculum"}}
+    autocomplete="off"
```

Comment:
I think this should be [name](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

```suggestion
    autocomplete="name"
```